api/worker.go

```diff
@@ -0,0 +1,539 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
```

Comment:
I believe this file was added by mistake?

Reply:
I think it was added by mistake!